src/blocks/block-devices/components/save.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import Device from './device';
+
+/**
+ * WordPress dependencies
+ */
+const { Component } = wp.element;
+
+export default class Save extends Component {
+	constructor() {
+		super( ...arguments );
+	}
+
+	render() {
+		return (
+
+			/* Save the block markup for the front end */
+			<Device { ...this.props }></Device>
```

Comment:
My editor is suggesting this be `<Device { ...this.props } />` since the body is empty.